### ISSUE TITLE
Support for other wiki languages than English

### ISSUE THREAD

**Issue by samuell** [rdfio/RDFIO#41]
We have some stupid hard-coded strings which might be different for different languages.

See this screenshot of code climate for hints on where to fix it:

![selection_296](https://user-images.githubusercontent.com/125003/28666494-d9559cb6-72c7-11e7-825d-bfe32e581ccb.png)

Looking into it.

**Comment by samuell** [comment]
Do @mwjames or @kghbln have some input on the most idiomatic way to access i18ed NSes?

**Comment by samuell** [comment]
I think I know the solution now:

```php
$titleWithNS = Title::makeTitleSafe( SMW_NS_PROPERTY, $theTitle );
```

... etc.

**Comment by samuell** [comment]
The changes are now implemented, as of cca71be1976a78f4a8f65d986e7fec35e382974a and fcd282346d839e448a1eceee8e943d1a148ed53f . Now just need to test this with another language somehow.

**Comment by samuell** [comment]
This is now tested manually, so closing (a CI would still be great).
